package.json

```diff
@@ -138,33 +146,94 @@
+   "sass:render:input": "sass-render --load-path=node_modules \"./components/input/src/**/*.css\" -t ./node_modules/@aurodesignsystem/auro-library/scripts/build/staticStyles-template.js",
+   "sass:render:dropdown": "sass-render --load-path=node_modules \"./components/dropdown/src/**/*.css\" -t ./node_modules/@aurodesignsystem/auro-library/scripts/build/staticStyles-template.js",
+   
+   "serve": "web-dev-server --open demo/ --node-resolve --watch",
+   "serve:input": "web-dev-server --open components/input/demo/ --node-resolve --watch --app-index \"./components/input/demo/index.html\"",
+   "serve:dropdown": "web-dev-server --open components/dropdown/demo/ --node-resolve --watch --app-index \"./components/dropdown/demo/index.html\"",
+
+   "build:input:full": "npm-run-all build:input build:api:input build:docs:input build:dev:assets:input",
+   "build:dropdown:full": "npm-run-all build:dropdown build:api:dropdown build:docs:dropdown build:dev:assets:dropdown",
+   
+   "build:input:release": "npm-run-all build:input:full build:test build:demo:input",
+   "build:dropdown:release": "npm-run-all build:dropdown:full build:test build:demo:dropdown",
+
+   "sweep": "rm -rf dist && find ./demo ./src ./components -not -path \"*/node_modules/*\" -type f \\( -name \"*.css\" -o -name \"*-css.js\" \\) -delete",
+
+   "test": "wtr --coverage",
+    "test:watch": "wtr --watch",
+    "test:input": "wtr \"./components/input/test/**/*.test.js\" --coverage",
+    "test:input:watch": "wtr \"./components/input/test/**/*.test.js\" --watch",
+    "test:dropdown": "wtr \"./components/dropdown/test/**/*.test.js\" --coverage",
+    "test:dropdown:watch": "wtr \"./components/dropdown/test/**/*.test.js\" --watch",
+
+    "linters": "npm-run-all scssLint esLint",
+    "scssLint": "stylelint \"./components/**/*.scss\"",
+    "esLint": "eslint 'components/**/*.js'",
+
+    "linters:input": "npm-run-all scssLint:input esLint:input",
+    "scssLint:input": "stylelint \"./components/input/**/*.scss\"",
+    "esLint:input": "eslint 'components/input/**/*.js'",
+
     "bundler": "rollup -c",
     "bundler:test": "rollup -c -w",
-    "scssLint": "stylelint \"./src/**/*.scss\"",
-    "dev": "concurrently --kill-others \"npm run build:watch\" \"npm run serve\"",
-    "dist:js": "copyfiles -u 1 -V './src/**/*.js' ./dist",
-    "esLint": "./node_modules/.bin/eslint src/**/*.js",
-    "linters": "npm-run-all scssLint esLint",
-    "preCommit": "node scripts/pre-commit.mjs",
-    "postCss:component": "node scripts/postCss.mjs",
-    "postinstall": "node packageScripts/postinstall.mjs",
-    "sass:render": "sass-render src/*.css -t ./scripts/staticStyles-template.js",
-    "serve": "web-dev-server --open demo/ --node-resolve --watch",
-    "sweep": "rm -rf ./demo/css ./dist | rm ./src/*.css ./src/*-css.js",
-    "test": "wtr --coverage",
-    "test:watch": "wtr --watch",
-    "types": "tsc",
-    "postversion": "npm-run-all build:api build:docs",
-    "prepare": "husky install"
```

Comment:
Is there a reason why you deleted all of these scripts? Are they going to be handled at the component level?

Reply:
Most of them are still in the file, but I changed their grouping to make it easier to read. However, 4 tasks were indeed missing, and I have returned them here:

https://github.com/AlaskaAirlines/auro-form/pull/16/commits/a7abf9a9feb2f8c7b2dac569af25e0eaaa0db04e